composer.json

```diff
@@ -12,7 +12,8 @@
         }
     ],
     "require": {
-        "php": ">=7.4"
+        "php": ">=7.4",
+        "ext-readline": "*"
```

Comment:
This should be moved to "require-dev". Since the Repl is just a tool for development and not for production.